### ISSUE TITLE
The player bar in the bottom doesn't show up keeping the memory of what I was just playing. When the app gets killed it just forgets what I was playing altogether.

### ISSUE THREAD

**Issue by PreyumKr** [vfsfitvnm/ViMusic#997]
### Steps to reproduce the bug

1. Play a song
2. Kill the app or simply close it completely
3. Open again 

### Expected behavior

The last song playback bar in the bottom with my last song and probably the last playback position.

### Actual behavior

A newly opened player with no records.

### ViMusic version

0.5.4

### Android version

Android 12

**Comment by PreyumKr** [comment]
> Do you have 'Persistent queue' enabled in Settings > Player?

Yes, and I noticed when I opened the app now it showed the last played song in the bottom. But when I remove it from recent apps and open again it doesn't show up again. There is something wrong in the process it works sometimes and not the others.

**Comment by vfsfitvnm** [comment]
Uhm, would you share a screen recording?

**Comment by ikanakova** [comment]
Could it be related to a #483?
